CONTRIBUTING.md

```diff
@@ -1,448 +1,137 @@
-- Generating a TypeScript declaration (`.d.ts`) file.
-
-To build the project:
-
-```bash
+# Build the project for production.
+# Run this command when you want to see the production version.
 npm run build
-```
-
-The CI system will try to build the project on each commit targeting the `master` branch. The CI check will fail if the build does.
 
-### Testing
-
-All projects have at least linting and unit tests. Linting is handled by [ESLint][eslint] with [Prettier][prettier] and unit tests are handled by [Vitest][vitest].
-
-To run all tests:
-
-```bash
-npm run test
-```
-
-If you'd like to run only the linter (note that it won't reformat your code; use the `format` script for that):
-
-```bash
+# Lint your changes before requesting a review. No errors are allowed.
 npm run lint
-```
+# Some errors can be fixed automatically:
+npm run lint -- --fix
 
-If you'd like to run only the unit tests:
+# Format your changes before requesting a review. No errors are allowed.
+npm run format
 
-```bash
+# Test your changes before requesting a review.
+# All changes should be tested. No failing tests are allowed.
+npm run test
+# Run only unit tests (optionally in watch mode):
 npm run unit
+npm run unit:watch
+# Run only type tests
+npm run types
 ```
 
-If you'd like to run only the unit tests in watch mode (re-runs tests each time a file is saved):
-
-```bash
-npm run unit -- --watch
-```
-
-When working on unit tests, you might want to update snapshots (be careful when doing so):
+## :building_construction: Submit a pull request
 
-```bash
-npm run unit -- --update-snapshots
-```
-
-The CI system will run tests on each commit targeting the `master` branch. The CI check will fail if any test does.
+> [!NOTE]
+> Code will be reviewed by the Prismic team before merging.[^3]
+>
+> Request a review by opening a pull request.
 
-### Publishing
+```sh
+# Open a pull request. This example uses the GitHub CLI.
+gh pr create
 
-> ⚠ &nbsp;Only project maintainers with at least collaborator access to the related npm package can publish new versions.
+# Someone from the Prismic team will review your work. This review will at
+# least consider the PR's general direction, code style, and test coverage.
 
-Publishing a package correctly involves multiple steps:
+# When ready, PRs should be merged using the "Squash and merge" option.
+```
 
-- Writing a changelog;
-- [Building](#building) the project;
-- Publishing a new version tag to GitHub;
-- Publishing build artifacts to [npm][npm].
+## :rocket: Publish
 
-In order to make sure all these steps are consistently and correctly done, we use [Standard Version][standard-version] and build scripts.
+> [!CAUTION]
+> Publishing is restricted to the Prismic team.[^4]
 
-To release a new version of the project:
+```sh
+# Checkout the master branch and pull the latest changes.
+git checkout master
+git pull
 
-```bash
+# Perform a dry-run and verify the output.
+# If it looks good, release a new # version.
```

Comment:
Not sure I understand `#` in `new # version` here and below 🤔

Reply:
Oops, it's a typo. It probably came from removing a line break that started with a `#`.